client/src/components/app.jsx

```diff
@@ -11,6 +14,9 @@ function App() {
   return (
     <div id="app">
       <Overview product={product} styles={styles} setStyles={setStyles} metaData={meta} />
+      <RelatedProducts />
```

Comment:
How are you getting the current product?

Reply:
For now I just hard-coded product_id inside of the Related Products because I was testing the axios requests and forgot to change it to your example.js.